src/corehaptics.cs

```diff
@@ -0,0 +1,394 @@
+//
+// CoreHaptics C# bindings
+//
+// Authors:
+//	Manuel de la Pena Saenz <mandel@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation All rights reserved.
+//
+
+using System;
+	
+using AVFoundation;
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+
+namespace CoreHaptics {
+
+#if MONOMAC
+	interface AVAudioSession {}
```

Comment:
add link to the open feedback/shadow issue

Reply:
yep, I was going to add it in a comment. Will add it in the source.